File: front_workspace/JS_workspace/day0514/js/Duck.js
class Duck extends Bird {
    // 의무!! 자식 클래스에서 생성자를 정의할 때 무조건 부모의 생성자 호출을 해야함
    constructor(color, age) {
        super(color, age); // super()가 부모의 생성자(constructor)를 의미
        console.log(this.color);
        console.log(this.age);
    }

    fly() {
        console.log("오리가 날아요");
    }
}

File: back_workspace/java_workspace/day0512/Duck.java
/*메서드 오버로딩(OverLoading) : 메서드 중첩기법
	1) 메서드의 이름 동일
	2) 매개변수의 자료형이나 갯수가 달라야함
*/
class Duck {
	//오리가 난다.
	public void fly(){

	}
	//오리가 조금 높은 높이에서 난다.
	public void fly(int height){
		System.out.println("오리가 조금 높은 높이에서 난다.");
	}

}